src/main/webapp/authentication-app/src/main-components/SignUpPage.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+import { Link } from "@reach/router";
+
+const SignUpPage = () => {
+    const [userEmail, setUserEmail] = useState("");
+    const [userPassword, setUserPassword] = useState("");
+    const [displayName, setDisplayName] = useState("");
+    const [error, setError] = useState(null);
+
+    const createUser = (event, userEmail, userPassword) => {
+        event.preventDefault();
+        setUserEmail("");
+        setUserPassword("");
+        setDisplayName("");
+    };
+
+    const onChangeHandler = (event) => {
+        const { name, value } = event.currentTarget;
+        if( name === userPassword) {
+            setUserPassword(value);
+        }
+        else if (name === displayName) {
+            setDisplayName(value);
+        }
+        else if (name === userEmail) {
+            setUserEmail(value);
+        }
+    };
+
+    return (
+        <div className="SignUp">
+        <h1 className="Title">Sign Up </h1>
+        <div className="SignUp-Area">
+          <form className="SignUp-form">
+            <label htmlFor="displayName" className="block">
+              Choose your Display Name:
+            </label>
+            <input
+              type="text"
+              className=""
+              name="displayName"
+              value={displayName}
+              placeholder= "E.x: AwesomeDude123"
+              id="displayName"
+              onChange={(event) => onChangeHandler(event)}
```

Comment:
Is display name like a username? I'm thinking either we replace this with a first name and last name input or add a first name and last name input inside the form. First and last name would be easier for contact purposes (messaging someone without having to ask for their name would be nice) and from the design doc I think we decided on giving each user a unique ID rather than using the username as the ID because it makes it easier to support features we may want in the future (username changes, etc)